bsip-0053.md

```diff
@@ -0,0 +1,264 @@
+    BSIP: 0053
+    Title: Blockchain scanning for inbound Stealth transactions
+    Authors: Christopher J. Sanborn
+    Status: Draft
+    Type: Protocol
```

Comment:
I think type "Informational" would be the correct choice here. While the proposal does describe a kind of client-interoperability protocol, the proposed changes are not mandatory for clients nor enforced in any way by the blockchain backend. Perhaps augmenting the type for clarification would be acceptable, although BSIP-0001 does not explicitly allow this.
`Type: Informational (Client Protocol)`